deploy/queue.ts

```diff
@@ -4,6 +4,7 @@ import {
     DEPLOY_PENDING_FILE_PATH
 } from "serverSettings"
 import { deploy } from "./deploy"
```

Comment:
It may be worth it to add a few simple unit tests to the queue as I could see more features being added in the future.

Reply:
Yeah good idea, will do!